Dockerfile

```diff
@@ -50,6 +54,14 @@ ARG COMMIT_SHA
 ENV SHA=${COMMIT_SHA}
 RUN echo "sha-${SHA}" > /etc/school-experience-sha
 
+# Create writable directories and set proper permissions for non-root user
+RUN mkdir -p /app/tmp /app/out /app/log /app/coverage /tmp /tmp/prometheus/ && \
+    chown -R appuser:appgroup /app /app/tmp /app/out /app/log /app/coverage /tmp /tmp/prometheus/ &&\
+    chmod -R u+rwX /app /app/tmp /app/out /app/log /app/coverage /tmp /tmp/prometheus/
```

Comment:
can we try this without /app in lines 59 and 60. That covers everything below it and is what we are trying to avoid